src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -113,7 +114,33 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
   }
 
   currentPageIsNotValid(): boolean {
-    return !this.pageValidationService.isPageValid(this.currentPage, this.editForm);
+    const data = this.editForm.controls.data['controls'];
+    if (data) {
+      if (this.currentPage.id === 'editAppealnewMatters') {
+        if (data.newMatters && data.hasNewMatters) {
+          const newMatters = data.newMatters.value ? data.newMatters.value : '';
+          const hasNewMatters = data.hasNewMatters.value ? data.hasNewMatters.value : '';
+          if (newMatters.length === 0 && hasNewMatters === 'Yes') {
+            return true;
+          }
+        }
+      } else {
+        if (data.email && data.mobileNumber) {
```

Comment:
This whole block now takes no account of how the CCD Case config is actually set up - I've seen configurations where there was an optional `email` and an optional `mobileNumber` field. This code will make that look invalid when it shouldn't.